docker-compose.gpu.yml

```
@@ -0,0 +1,76 @@
+      - AUDIOCRAFT_DEFAULT_JASCO_MODEL=facebook/jasco-chords-drums-melody-1B
+      - NVIDIA_VISIBLE_DEVICES=all
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: 1
+              capabilities: [gpu]
+        limits:
+          memory: 32G
+        reservations:
```

YAMLlint (1.37.1)

[error] duplication of key "reservations" in mapping

(key-duplicates)